gameplay/Gameplay.gd

```diff
@@ -6,14 +6,28 @@ extends Control
 var mock_goal = 4
 var mock_victory = false
 
-# Called when the node enters the scene tree for the first time.
+var _backpack_initialized = false
+var _columns_ready_count = 0
+var _expected_columns_initialized_count = 3
+
 func _ready():
 	database.reset_values()
 	_set_mock_goal()
 
-# Called every frame. 'delta' is the elapsed time since the previous frame.
 func _process(delta):
-	pass
+	if (_are_columns_rendered()):
+		_backpack_initialized = true
+		$Backpack.snap_position_to_column($Columns/GameplayColumn)
+		$Backpack.visible = true
+		
+func _are_columns_rendered():
+	return (
+		not _backpack_initialized
+		and _columns_ready_count == _expected_columns_initialized_count
+	)
+
+func _increment_columns_ready_count():
+	_columns_ready_count += 1
```

Comment:
This "is it time to snap the backpack to its correct position" check should hopefully be able to work in a different way. 

The point of the _ready method is it should only be called after all child nodes have finished initializing. 

I think I remember that it didn't work well when we were doing initial testing before, but I'm interested in finding a more straightforward way to determine "after columns finished sorting themselves out"

Reply:
I found the answer: the Columns container emits an event (`sort_children`) when it finishes positioning and sizing its contents.

We can use that to snap the backpack to a column whenever the number of columns (or their position, I guess) changes.

Reply:
Fixed by merging your PR, thank you for figuring out the right event to listen for.